app.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express")
+const mongoose = require("mongoose")
+const dotenv = require("dotenv")
+const app = express()
+dotenv.config()
+
+mongoose.connect(process.env.MONGODB_URI).then(() => console.log("connected to database"))
+
+const itemsSchema = new mongoose.Schema({
+    name: String,
+    age: Number,
+})
```

Comment:
kindly consider giving the schema all all the fields required as mentioned below 
- id: a unique identifier for the item.
- name: the name of the item.
- description: a description of the item.
- price: the price of the item.